### PR TITLE
Update CONTRIBUTING.md for main branch

### PR DESCRIPTION
**Pull request by backwind1233**
As title

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,44 +1,44 @@
...
+          -Drevapi.skip=true -Djacoco.skip=true \
+          -Dparallel-test-playback \
+          -Pdev \
+          -f sdk/spring/pom.xml
+   ```
+5. Make pull request to merge the branch into `main`.
+
+      > There are github actions to check some status here.
+
+### 02. Contribute to other branch except
```
Comment (saragluna) [inline CONTRIBUTING.md:35]: to other branches or to other branches except main

Comment (saragluna) [reply CONTRIBUTING.md:35]: We'd better also update the readme file to explain our branch policy

**Review by saragluna — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (backwind1233) [reply CONTRIBUTING.md:35]: Ok

**Review by saragluna — APPROVED** [review]
LGTM

**Pull request merged**